the-event-loop/interval.js

```diff
@@ -0,0 +1,11 @@
+var count = 3;
+const timer = () => {
+  if (count === 0) {
```

Comment:
(Optional) Sometimes it's best to check `<= 0` instead of `=== 0` since if `count` ever goes negative this will loop forever.